src/nupic/utils/LoggingException.cpp

```diff
@@ -34,7 +34,11 @@ LoggingException::~LoggingException() throw()
   if (!alreadyLogged_) {
     // Let LogItem do the work for us. This code is a bit complex
     // because LogItem was designed to be used from a logging macro
+#if defined(NTA_OS_WINDOWS) && defined(NTA_COMPILER_MSVC) && defined(NDEBUG)
+    auto li = new LogItem(filename_.c_str(), lineno_, LogItem::info);
```

Comment:
This was to cope with the movement to Google unit testing. On AppVeyor the tests would halt upon failure, so this was placed here for the specific version AppVeyor created (Windows MSVC Release build). It'd be good to reverse this and address any issue with tests stopping CI elsewhere.